Sources/NSAttributedStringExtensions.swift

```diff
@@ -73,5 +73,12 @@ public func += (left: inout NSAttributedString, right: NSAttributedString) {
     ns.append(right)
     left = ns
 }
+    
+/// EZSE: Sum of one NSAttributedString with another NSAttributedString
+public func + (left: NSAttributedString, right: NSAttributedString) -> NSAttributedString {
```

Comment:
Does NSAttributedString support any other operators by default. If yes, then what is the operator precedence of those operators wrt to this +?

Reply:
I added one more test. What do you think?

Reply:
yea, this verifies my assumption. I will approve as soon as it passes the build.